acct-user/amule/amule-0.ebuild

```diff
@@ -0,0 +1,13 @@
+# Copyright 2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit acct-user
+
+DESCRIPTION="User for net-p2p/amule"
+ACCT_USER_ID=468
+ACCT_USER_GROUPS=( amule )
+ACCT_USER_HOME="/var/lib/amule"
```

Comment:
Please verify whether this is needed or not, per new instructions in https://devmanual.gentoo.org/ebuild-writing/users-and-groups/index.html#choosing-a-home-directory

Reply:
I moved home directory management into net-p2p/amule ebuild. Init sctipt sets ${HOME} environment variable that should be enough for the daemon to work properly.